src/game/scenes/mainmenu/menu_video.c

```diff
@@ -128,6 +133,18 @@ component *menu_video_create(scene *s) {
     menu_attach(menu, label_create(&tconf, "VIDEO"));
     menu_attach(menu, filler_create());
 
+    // Renderer selector
+    component *renderer_selector =
+        textselector_create(&tconf, "API:", "Choose the video renderer API to use", renderer_toggled, local);
```

Comment:
this textselector will always return to the first item in its list (when leaving the video options & going back), as there's no `textselector_set_pos` call.

Reply:
https://github.com/omf2097/openomf/pull/775